### ISSUE TITLE
为什么使用proto2

### ISSUE THREAD

**Issue by jiejieTop** [daohu527/pb_rosmsg#1]
如题！

**Comment by daohu527** [comment]
because apollo use proto2, in order to maintain compatibility.

Actually eventually I want to port some tools from ros to apollo, the message is one of the most important parts of it, I will migrate some slam algorithms first

**Comment by jiejieTop** [comment]
我看Apollo也是在用2，就很奇怪~3不好吗

**Comment by daohu527** [comment]
yeah, use proto2 is a conservative strategy, especially for compatibility
